apps/platform-app/prisma/migrations/20241203175027_change_slug_to_required/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Made the column `slug` on table `Profile` required. This step will fail if there are existing NULL values in that column.
```

Comment:
**issue**: Ensure there are no NULL values in the `slug` column before applying this migration to prevent failures.